app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/models/Allergen.java

```diff
@@ -0,0 +1,33 @@
+package com.epfl.neighborfood.neighborfoodandroid.models;
+
+import com.epfl.neighborfood.neighborfoodandroid.R;
+
+public enum Allergen {
+    CELERY (R.drawable.celery, "celery"),
+    MILK (R.drawable.milk, "milk"),
+    FISH (R.drawable.fish, "fish"),
+    CHEESE (R.drawable.cheese, "cheese"),
+    GLUTEN (R.drawable.gluten, "gluten"),
+    HONEY (R.drawable.honey, "honey"),
+    LOBSTER (R.drawable.lobster, "Lobster"),
+    SOY (R.drawable.soy, "Soy"),
+    EGGS (R.drawable.eggs, "Eggs"),
+    CHOCOLATE (R.drawable.chocolate, "Chocolate");
```

Comment:
I like the fact you added an enum for the allergens this will help with the feature place meal I implemented before(maybe next sprint ).